bpm/bonita-core/bonita-process-engine/src/main/java/org/bonitasoft/engine/api/impl/transaction/actor/ImportActorMapping.java

```diff
@@ -10,22 +10,24 @@
  * You should have received a copy of the GNU Lesser General Public License along with this
  * program; if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth
  * Floor, Boston, MA 02110-1301, USA.
- **/
+ ******************************************************************************/
+
 package org.bonitasoft.engine.api.impl.transaction.actor;
 
 import java.io.IOException;
-import java.io.StringReader;
 import java.util.List;
 import java.util.Set;
 
+import javax.xml.bind.JAXBException;
```

Comment:
Throwing BonitaExceptions here imply to (actually pretty extensively) rework the dependecies of the project, since the classes those Exceptions are passed to classes located in modules not actually able to see them. So it's possible to do it, but IMHO not worth it, since the JAXBException(s) are systematically caught by the class calling the methods of the converter, and therefor never leave the engine